languages/en.ini

```
@@ -353,6 +358,10 @@ delete_selected_favorites = "Delete Selected Saved Items"
 delete_tag = "Delete Tag"
 delete_tags = "Delete Tags"
 delete_tags_by = "Delete Tags By"
+delivery_request = "Request Delivery"
+delivery_request_distance = "Distance Delivery Services"
+delivery_request_distance_text = "We deliver materials directly to your home."
```

This might be a more general message, since if the user has an address on file, it might or might not be to their home.

```suggestion
delivery_request_distance_text = "We deliver materials directly to your address."
```